packages/cli/commands/project/dev.js

```diff
@@ -76,45 +71,32 @@ exports.handler = async options => {
   const accounts = getConfigAccounts();
   let targetAccountId = options.accountId;
   let createNewSandbox = false;
-  let chooseNonSandbox = false;
+  const defaultAccountIsSandbox = isSandbox(accountConfig);
+
+  if (!targetAccountId && defaultAccountIsSandbox) {
+    targetAccountId = accountId;
+  }
 
   if (!targetAccountId) {
+    logger.log(
+      uiLink(i18n(`${i18nKey}.logs.learnMoreLink`), 'https://www.google.com')
```

Comment:
I would suggest flagging this with a `TODO` to avoid any heartbreak.

Reply:
Good call. I may actually delete this for now because I don't think we have a doc setup for this yet.